paws_collector.js

```diff
@@ -523,7 +523,14 @@ class PawsCollector extends AlAwsCollector {
                     });
                     await Promise.all(promises);
                     return callback(null, privCollectorState, nextInvocationTimeout);
-                } else {
+                } else if (err.statusCode && err.statusCode === 307 && process.env.paws_collection_interval && process.env.paws_collection_interval > 0) {
+                    const paws_collection_interval = Math.floor(process.env.paws_collection_interval / 2);
```

Comment:
I think we should introduce some lower bound for `paws_collection_interval`.  If `process.env.paws_collection_interval` is 1 when it hits this clause, we would end up with a new collection interval of 0
```javascript
> process.env.paws_collection_interval = 1
1
> Math.floor(process.env.paws_collection_interval / 2)
0
```